pom.xml

```diff
@@ -234,6 +231,11 @@
                 <![CDATA[
                   {
                     "revapi" : {
+                      "semver": {
+                        "ignore": {
+                          "enabled": true
```

Comment:
nice, does it mean the plugin will automatically increase the minor version if there is an incompatible change? (because we are < 0.x.y)

Why the descriptions are not needed anymore? Is it because of the `< 0.x.y` thing?